devicetypes/smartthings/zigbee-window-shade.src/zigbee-window-shade.groovy

```diff
@@ -82,9 +83,20 @@ private List<Map> collectAttributes(Map descMap) {
 	return descMaps
 }
 
+def installed() {
+	log.debug "installed"
+	sendEvent(name: "supportedWindowShadeCommands", value: JsonOutput.toJson(["open", "close", "pause"]))
+}
+
 // Parse incoming device messages to generate events
 def parse(String description) {
 	log.debug "description:- ${description}"
+
+	// update existing devices
+	if (!state.supportedWindowShadeCommands) {
```

Comment:
This will need to be ` if (!device.currentValue("supportedWindowShadeCommands"))`

Reply:
instead of  
`if (!state.supportedWindowShadeCommands) ` ?